### DISCUSSION TITLE
2023-07-04

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#6] [daily_meeting]
### 어제의 회고
P:
체크인, 체크아웃을 할 때 사용하는 플랫폼(깃헙, 노션, 피그잼)이 동시 다발적으로 작성되고 있어 흐름을 놓칠 때가 있다.
여전히 새로운 아이디가, 플로우가 등장하는데 이를 어떻게 표현해야 할까?

T:
병렬적으로 작성하지 않거나, 서기가 한 번에 작성해주면 좋을듯?
각자의 생각을 발표하는 시간 가지기

### 오늘의 할 일

- 프론트엔드
    - ~~와이어 프레임 스케치 마무리~~ 
- 공통
   - ~~새로운 의견에 대한 논의(스터디 모집 시점에 대한 논의)~~
   - ~~IA 공유, 와이어 프레임 스케치 공유 후 MVP 명세 픽스~~
   - ~~솔라와 커피챗~~
   - ~~사용자 스토리 제출~~
   - ~~솔라 커피챗 전 현재 고려사항 정리해서 질문지 보내두기~~

### 내일의 할 일
- 공통
  - 도메인 용어 사전 완성하기
- 프론트엔드
  - 웹팩 설정
  - wiki 기술 스택(styled-component 선정 이유) 정리
  - 와이어 프레임 마무리 조금하기
- 백엔드
  - 개발문서 마무리(기술 스택 및 선정 이유 정리)
  - API 명세

**Comment by nlom0218** [comment]
### 체크인

10 - 날씨가 뜨겁지 않아 좋다~~

### 체크아웃

9 - 어제보다 더 명확해진 서비스의 방향이 좋았다. 대신 비가 너무 온다 😭

### KPT

K: 자유로운 의견 제시
P: 기능에 대한 의견이 아닌 진행 방식에 대한 의견으로 시간을 많이 소비한 것이 아쉬웠다.
T: 진행자에게 확실한 권한을 주면 어떨까?

**Comment by jaehee329** [comment]
### 체크인

8 - 이제야 화요일..? 조금 피곤하네요 두어시간 더 자고 싶다

### 체크아웃

8 - 눈이 피곤해요. 근데 할 건 많이 해서 다행이야

### KPT

K: 중간에 리프레시 겸 무중력 광장에서 대화하는 것 좋았다.
P: 
T: 환기가 필요할 때 누군가 무중력 광장에서 이야기하자고 제안해보자.

**Comment by woosung1223** [comment]
### 체크인
9 - 할일이 너무 많아서 오히려 좋아

### 체크아웃
9점 - 오늘 할 일 다해서 기분 조와용

### KPT
K: 이전에 선정했던 Action Item에 대해 사용하려고 노력한 것.
P: 하루종일 기획, 기술 관련 이야기만 한 것 같아서 지치는 면도 있다!
T: 중간중간에 잡담을 5분씩이라도 섞어보면 어떨까요?

**Comment by aak2075** [comment]
### 체크인

10점 - 10점이라고 생각하고 하루를 파이팅 있게 시작해 봅시다

### 체크아웃

10점 - 오늘 할 일 다 끝낸게 매우 만족스럽다.

### KPT

K - 오늘 할 일 모두 끝난것. 
P - 회의 시간이 50분이 넘어가는 경우가 있다
T - 타이머 활용

**Comment by woo-jk** [comment]
### 체크인
10 - 점수 깎일 이유가 없어서 만점~

### 체크아웃
10 - 우리의 서비스 MVP에 대해서 확실하게 구체화된 것 같아서 좋다잉

### KPT
K: 무중력 광장 회의 좋다.
P: 회의 중 의견 제시에 대해 약간 부담이 있다.
T: 회의 중 의견 제시에 더 편한 분위기가 조성되면 좋을 듯하다.

**Comment by yeopto** [comment]
### 체크인

10점 - 10점이라고 마인드 컨트롤해야 하루가 활기차진다.

### 체크아웃

10점 - 오늘 할 일을 다 끝내서 뿌듯하다.

### KPT

K: 논의가 길어지면 리프레쉬할겸 무중력광장에서 회의하는 방식이 괜찮은 것 같다.
P: 없음.
T: 없음.

**Comment by MoonJeWoong** [comment]
### 체크인
8 - 오늘 아침에 공지에 올라온 퀴즈를 맞춰서 기분이 좋네용! 2점은 살짝 졸려서 깎았습니다.

### 체크아웃
10 - 오늘 계획한 할 일을 모두 끝낸 첫 날이라 기분이 많이 좋습니다. 고생 많으셨어요!!

### KPT
K: 분위기 환기를 위해 무중력 광장으로 이동해서 얘기한 부분이 좋았다.
P: 중간 중간 논의에서 힘 빠지는 순간에 잘 대처하지 못한 것 같다.
T: 개발과 전혀 상관 없는 잡담을 한다. (오늘 정말 효과가 좋았다!)
